get.sh

```diff
@@ -69,6 +69,12 @@ getPackage() {
 
     if [ "$?" = "0" ]; then
 
+    tar -xvzf $targetFile
```

Comment:
Let's check the exit code to make sure it worked.

Reply:
Okay 😄

Reply:
Need this change please.